libsolidity/formal/Why3Translator.cpp

```diff
@@ -757,6 +757,15 @@ bool Why3Translator::visit(Literal const& _literal)
 	return false;
 }
 
+bool Why3Translator::visit(PragmaDirective const& _pragma)
+{
+	auto semVerMatch = _pragma.parseSemVer();
```

Comment:
As per the comments above, it is probably fine to check only the first identifier. You could add a helper function to `PragmaDirective` that returns the first identifier as a string.